### ISSUE TITLE
[2021/03/04] Machine Learning 輪講

### ISSUE THREAD

**Issue by agatan** [wantedly/machine-learning-round-table#89]
## Why

Machine Learning 輪講は最新の技術や論文を追うことで、エンジニアが「技術で解決できること」のレベルをあげていくことを目的にした会です。

prev. #87 

## What

話したいことがある人はここにコメントしましょう！
面白いものを見つけた時点でとりあえず話すという宣言だけでもしましょう！

**Comment by agatan** [comment]
## LambdaNetworks: Modeling long-range Interactions without Attention

- ICLR 2021 spotlight paper
- https://arxiv.org/abs/2102.08602

**Comment by agatan** [comment]
## Long Range Arena: A Benchmark for Efficient Transformers

- https://arxiv.org/abs/2011.04006
- https://ai-scholar.tech/articles/transformer/long-range-arena

**Comment by agatan** [comment]
## GradInit: Learning to Initialize Neural Networks for Stable and Efficient Training

- https://arxiv.org/abs/2102.08098v1

**Comment by hiroto0227** [comment]
## Jointly Leveraging Intent and Interaction Signals to Predict User Satisfaction with Slate Recommendations

https://github.com/hiroto0227/paper/issues/2

- https://dl.acm.org/doi/pdf/10.1145/3308558.3313613
- WWW2019で発表されたspotifyの論文
- Spotifyのホーム画面では、ユーザーのintentが複数存在していて、そのintentごとに推薦の満足度(satisfaction)は異なる。実際にintentを用いてsatisfactionを予測すると、その予測精度は向上した。
  - 例えば、「自分の好きな音楽を簡単に聞きたい」「いろいろな音楽を聞いて新しい音楽を発掘したい」というintentがある。
- satisfactionを最大化するためには、userのintentを理解し予測する必要があり、その予測したintentを用いて推薦を行うと良いのではないかと結論づけている。

**Comment by zerebom** [comment]
## Improving Pairwise Ranking for Multi-label Image Classification
src: https://arxiv.org/pdf/1704.03135.pdf
pytorch impl: https://gist.github.com/NegatioN/eb2c23fc98e67a6396f6ea77e384c549

![image](https://user-images.githubusercontent.com/38466611/109943111-2fd35c00-7d18-11eb-88a6-bd6e3d922f6e.png)


MultiLabelのRankingタスクで使用されるNN用の損失関数。
rainforestコンペ 5th solutionsで使用されていた。

Label Prediction, Decisionの最適化を同時に行う。

![image](https://user-images.githubusercontent.com/38466611/109942841-f0a50b00-7d17-11eb-93d7-2b120a969eb4.png)


詳細: https://github.com/zerebom/paper-books/issues/46
